src/components/ec-inline-input-field/ec-inline-input-field.vue

```diff
@@ -71,6 +83,11 @@ export default {
       valueForLoading: this.value,
     };
   },
+  computed: {
+    isCopyingAllowed() {
+      return this.isCopiable && !this.isEditable;
```

Comment:
I think this is redundant, `isCopyingAllowed` is being used in an else from `isEditable` by definition it will always be  `!isEditable` https://github.com/Ebury/chameleon/pull/361/files#diff-9eb3042a3cc3cf639dc34293be9d9657R7

Hence, you can use the else with `isCopiable` prop directly and remove this computed method, unless I'm missing something

Reply:
You right amended, doesn't make sense on an v-else-if